### PR TITLE
Force browsers not to use cached version of login page

### LINKED ISSUE #37: Occasional JSF issue when going through the JWT guide
Sometimes there is something bad cached in the browser such that a JSF exception occurs when you try to login to the frontendUI.
 
The message is (something like) `No such view state found`. Exception was on customer machine but @Emily-Jiang has seen it too.

Switching to a private browser window (i.e. no cache or cookies) solves the problem.

### PR DESCRIPTION
**Pull request by proubatsis**
Fixes #37

### CODE REVIEW

**Comment by evelinec** [comment]
@wtlucy Can you help us review these changes for a JSF front end page?

File: finish/frontendUI/src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -2,6 +2,23 @@
 <web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"
     version="3.1">
+
```
[the remark this answers is not in the document]
Comment (proubatsis) [reply finish/frontendUI/src/main/webapp/WEB-INF/web.xml]: It was already in there, I just moved it to the top of the file.

**Comment by evelinec** [comment]
@yeekangc would you help take a look at this PR? Thanks.

**Comment by yeekangc** [comment]
For addressing the issues that we encountered, the fixes are fine.  We need to look into what more we should consider for state saving and JSF in general but they are separate topics.

**Pull request merged**